src/main/java/com/programmers/ticketparis/controller/performance/PerformanceController.java

```diff
@@ -0,0 +1,52 @@
+package com.programmers.ticketparis.controller.performance;
+
+import com.programmers.ticketparis.dto.performance.request.PerformanceCreateRequest;
+import com.programmers.ticketparis.dto.performance.request.PerformanceUpdateRequest;
+import com.programmers.ticketparis.dto.performance.response.PerformanceResponse;
+import com.programmers.ticketparis.service.performance.PerformanceService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/performances")
+public class PerformanceController {
+    private final PerformanceService performanceService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public void createPerformance(@RequestBody PerformanceCreateRequest createRequest) {
```

Comment:
`p1`
valid 어노테이션이 빠졌어요~
다른곳도 확인해주세여

```suggestion
    public void createPerformance(@Valid @RequestBody PerformanceCreateRequest createRequest) {
```

Reply:
넵! 반영하여 수정하였습니다.
반영: 3d82dd9e6093837b569e0b149159e502f5276a04